### DISCUSSION TITLE
Updater bot triggers?

### DISCUSSION THREAD

**Discussion by lazytarget** [alexbelgium/hassio-addons#1649] [General]
Hi!
I am currently waiting on a new release of the Firefly addon, since they have fixed an issue that occurs in my system. I see that you use a "updater bot" for updating any existing addons. My question is could we trigger a new build for this new version to be compiled and also is is there a way to automate the updater bot by triggering for example on a nightly schedule?

**Comment by alexbelgium** [comment]
Hi sure, auto updates are on Saturday but I can trigger it manually I'll do it now

**Reply by alexbelgium** [reply]
I do it on a weekly and not nightly anymore to have all issues during weekend and not work week, and also as I noticed that it left time for invalid upstream containers to be pulled back (happened a couple time in the past).

**Reply by lazytarget** [reply]
Wonderful! Ok, I see.
Weekly builds should actually suffice and like in this case, can be run explicitly when needed.

Thank you!
